src/extra/components/dld.py

```diff
@@ -357,7 +357,7 @@ def signals(self, pulse_dim='pulseId', extra_columns={}, max_method=None):
         This data is primarily for detector diagnostics purposes and
         should generally not be used for scientific data analysis,
         please refer instead to reconstructed x, y, t data obtained from
-        [hits()](extra.components.DelayLineDetector.hits()).
```

Comment:
Could you please rebase? 🙈 

I fixed these issues specifically in 117e7b38a5715880880dc93fbdab58e5d7286bb7 earlier.

Reply:
Ah yes you're right, I missed that... I've rebased it now.